src/test/resources/features/foi/Allocation.feature

```diff
@@ -0,0 +1,18 @@
+@FOIAllocation @FOI @FOIWorkflow
+Feature: Allocation
+
+  Background:
+    Given I am logged into "CS" as user "FOI_USER"
+    When I create a "FOI" case and move it to the "Allocation" stage
+    And I load and claim the current case
+
+  #HOCS-3626 #HOCS-3268 #HOCS-2326
+  @FOIRegression
+  Scenario: User is able to complete the Allocation stage
+    And the Requested Question should be displayed in the summary tab
```

Comment:
why is this assertion at the start of this scenario?
should it be its own scenario? or is the step just worded so it doesnt make sense in this script (maybe reword it?)

Reply:
Yeh I think this should be 2 scenarios. The assertion at the start has nothing to do with the scenario title